designer/client/src/components/toolbarSettings/types.ts

```diff
@@ -1,5 +1,5 @@
 import { ToolbarsSide } from "../../reducers/toolbars";
-import { ButtonsVariant } from "../toolbarComponents/ToolbarButtons";
+import { ButtonsVariant } from "../toolbarComponents/toolbarButtons/index";
```

Comment:
Something is wrong with this import toolbarButtons/index it should be just toolbarButtons

Reply:
fix everywhere